src/app/components/repeater/index.html

```diff
@@ -89,7 +98,15 @@
     >
     Indicates whether to display an inline form within the repeater. Users can toggle between displaying and hiding the inline form. This property accepts <sky-code>boolean</sky-code> values.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="tag"
+      defaultValue="-1"
```

Comment:
There is no default value for `tag`. It's just `undefined`. Also, it accepts `any` values, not `number`.

Reply:
Sorry. I got my wires crossed and was looking at `tabIndex` in the code. I'll update. Thanks!